modules/gdscript/gdscript_parser.h

```diff
@@ -42,6 +42,26 @@ struct GDScriptWarning;
 
 class GDScriptParser {
 public:
+	struct IndentLevel {
+		int indent;
+		int tabs;
+
+		bool is_mixed(IndentLevel other) {
+			return (
+					(indent == other.indent && tabs != other.tabs) ||
+					(indent > other.indent && tabs < other.tabs) ||
+					(indent < other.indent && tabs > other.tabs));
+		}
+
+		IndentLevel() :
+				indent(0),
+				tabs(0) {}
```

Comment:
You can initialize directly upon declaration.

Reply:
Ah well this is the `3.2` branch so it's not so important, we hadn't ported the codebase yet.